pkg/plugin/view/configaudit/report_view.go

```diff
@@ -4,36 +4,66 @@ import (
 	"fmt"
 	"sort"
 	"strconv"
+	"strings"
+
+	"github.com/aquasecurity/starboard/pkg/kube"
 
 	"github.com/aquasecurity/starboard-octant-plugin/pkg/plugin/view"
 	starboard "github.com/aquasecurity/starboard/pkg/apis/aquasecurity/v1alpha1"
 	"github.com/vmware-tanzu/octant/pkg/view/component"
 )
 
-func NewReport(report *starboard.ConfigAuditReport) (flexLayout *component.FlexLayout) {
+func NewReport(workload kube.Object, configAuditReportsDefined bool, report *starboard.ConfigAuditReport) (flexLayout *component.FlexLayout) {
 	flexLayout = component.NewFlexLayout("")
+
 	flexLayout.AddSections(component.FlexLayoutSection{
 		{
 			Width: component.WidthFull,
 			View:  component.NewMarkdownText("#### Config Audit Reports"),
 		},
 	})
 
+	if !configAuditReportsDefined {
+		flexLayout.AddSections(component.FlexLayoutSection{
+			{
+				Width: component.WidthFull,
+				View: component.NewMarkdownText(fmt.Sprintf(
+					"The `%[1]s` resource, which represents config audit reports is not defined.\n"+
+						"> You can define such resource by running the [Starboard CLI][starboard-cli] init command:\n"+
```

Comment:
```suggestion
						"> You can define this resource by running the [Starboard CLI][starboard-cli] init command:\n"+
```

Reply:
I'm wondering if "You can create this resource definition" would be even better, wdyt?

Reply:
Yes. I like it and I applied it here and for vulnerability report